src/scripts.js

```diff
@@ -96,15 +125,34 @@ function cardButtonConditionals(event) {
   if (event.target.classList.contains('favorite')) {
     favoriteCard(event);
   }
-  if (event.target.classList.contains('card-picture')) {
+  if (event.target.classList.contains('instruction')) {
+    declareRecipe(event)
     findDirections(event);
   } 
-  if(event.target.classList.contains('add-button')){
-    // console.log(typeof event.target.id)
+  if(event.target.classList.contains('add-button')) {
+    console.log(event)
     addToCookLater(event);
   }
- }
-
+  if(event.target.classList.contains('icon')) {
+    let id = event.target.id
+    let currentRecipe = cookbook.recipes.find(x => x.id === +id)
+    let ingredientFeedback = Object.values(userPantry.giveFeedbackOnIngredients(currentRecipe, cookbook))
+    domUpdates.displayIngredientFeedback(ingredientFeedback, id)
+  }
+  if(event.target.classList.contains('back-button')){
+    domUpdates.populateCards(cookbook.recipes, cardArea, currentFavs())
+  }
+  if(event.target.classList.contains('purchase-button')){
```

Comment:
As mentioned before I think we should change the name of the 'button' `purchase` to just `add` since we are not tracking the user's wallet